graphql/database/groupImpact/updateGroupImpactMetric.js

```diff
@@ -98,15 +107,34 @@ const updateGroupImpactMetric = async (userContext, causeId) => {
   }
 
   // Now update GroupImpactMetric
-  const newDollarProgress = Math.round(
-    groupImpactMetric.dollarProgress + 10 ** 6 * getEstimatedMoneyRaisedPerTab()
-  )
+  let newDollarProgress = 0.0
+  let { dollarProgressFromTab } = groupImpactMetric
```

Comment:
Nit, you can combine these into one statement. I'll be digging around in this code soon so I can take a crack at it later.

Reply:
@jedtan Maybe. I think newDollarProgress is used as a temp variable then merged back into groupImpactMetric. Yeah. maybe it can be refactored to be cleaner. Feel free to take a crack at it. Thanks!